data/data/manifests/openshift/baremetal-provisioning-config.yaml.template

```diff
@@ -9,3 +9,4 @@ spec:
   provisioningNetwork: "{{.Baremetal.ProvisioningNetwork}}"
   provisioningDHCPRange: "{{.Baremetal.ProvisioningDHCPRange}}"
   provisioningOSDownloadURL: "{{.ProvisioningOSDownloadURL}}"
+  watchAllNamespaces: false
```

Comment:
What is are we gaining by explicitly setting this value?

Reply:
You are correct in that we don't need to have this patch.  My thinking was that it would provide documentation of the fields available and the value which we expect.  Otherwise if you follow the path through the system you just magically see watchAllNamespaces show up in the CR.

Reply:
The manifests are not intended to be documentation. I would prefer to see only the necessary fields specified, but I do not care too much if the baremetal team feels it is better to include it.

Reply:
If we only ever envisage this option being set day-2 then I tend to agree with @staebler - that case would be covered by a user looking at the CRD and/or by actual documentation.

However I suspect in some cases users may want to modify this manifest at install time via `create manifests`, in which case having this option in the manifest makes it marginally more obvious, and could help avoid mistakes like typos in the field name.

Reply:
Well I'm all about consensus.  This was mostly just my hair-brained idea (except I don't have hair).  If you think we don't need it I'll close it.